dangerfile.js

```diff
@@ -79,3 +79,59 @@ if (!isDependabotPR) {
     // Check Reviewer 2
     checkReviewerSection(reviewer2Section ? reviewer2Section[0] : null, 'Reviewer 2');
 }
+
+// README checks for Web Components (excluding core/testing projects)
```

Comment:
Might be worth creating a ticket to finally start splitting the danger checks into separate importable functions, before it's too late 😄

Reply:
yeah can do!